packages/runtime/src/core.ts

```diff
@@ -56,6 +46,103 @@ interface LoadRemoteMatch {
   remoteSnapshot?: ModuleInfo;
 }
 
+export async function getRemoteModuleAndOptions(options: {
```

Comment:
Will this still be exported for use?

Reply:
This method should be placed inside the remote handle and let the shared handler call the remote handler

Reply:
So its hook can also be placed in remote